src/System.Management.Automation/resources/ParserStrings.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
```

Comment:
Let's not change the unrelated part of the file.

Reply:
Fixed.